### PR TITLE
Improve Docs dataset layout issues

### PR DESCRIPTION
**Pull request by jk4e**
Fixed and improved the layout of some `datasets` pages in `docs` to make them more consistent overall. Thanks!

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Improved the documentation for dataset usage and citations to enhance clarity and user experience.

### 📊 Key Changes
- Reformatted the citation examples to use collapsible sections for better readability and organization.
- Added `Python` and `CLI` tabs to code examples for training models on various datasets.

### 🎯 Purpose & Impact
- **Improved Readability**: Clearer layout and formatting make it easier to understand how to use datasets and cite them properly. 📚
- **User-Friendly Examples**: Providing both Python and CLI examples helps users follow best practices regardless of their preferred method. 👩‍💻
- **Proper Attribution**: Ensures correct and easy citation of datasets, supporting continued research and development. 📜

### CODE REVIEW

**Review by ambitious-octopus — CHANGES_REQUESTED** [review]
Hey @jk4e, Thanks for the PR great fixes! Can you please take a look at the CI issues?

File: docs/en/datasets/classify/imagewoof.md [hunk 1]
```diff
@@ -59,18 +59,22 @@ ImageWoof dataset comes in three different sizes to accommodate various research
 
 To use these variants in your training, simply replace 'imagewoof' in the dataset argument with 'imagewoof320' or 'imagewoof160'. For example:
 
-```python
-from ultralytics import YOLO
+!!! Example "Example"
 
-# Load a model
-model = YOLO("yolov8n-cls.pt")  # load a pretrained model (recommended for training)
+    === "Python"
```
Comment (ambitious-octopus) [inline docs/en/datasets/classify/imagewoof.md:64]: To keep consistency with the example above I would also add an example with the Ultralytics Command Line Interface here.

**Review by ambitious-octopus — CHANGES_REQUESTED** [review]

File: docs/en/datasets/classify/imagewoof.md (hunk 1 above)
Comment (glenn-jocher) [reply docs/en/datasets/classify/imagewoof.md:64]: Added CLI!

**Comment by glenn-jocher** [comment]
@jk4e PR merged!! Thank you for these fixes. Dataset consistency is super important to help us stay organized and help the users understand the docs better. 

If you spot any other areas for improvement please submit a new PR. Thank you for your contributions!

**Pull request merged**